test/queries/addUser.test.js

```diff
@@ -0,0 +1,30 @@
+const tape = require('tape');
+const dbBulid = require('../../server/database/config/dbbuild');
+const { users: { getUserByUsername, addUser } } = require('../../server/database/queries');
+
+tape('test for adduser query', (t) => {
+  const expected = [{
+    id: 5,
+    username: 'fadi1',
+    email: 'fadi1@gmail.com',
+    password: 'asd',
+    isactive: true,
+    bio: null,
+    avatar_path: null,
+    native_lang_id: 1,
+    learning_lang_id: 1,
+  }];
+  dbBulid()
+    .then(() => addUser({
```

Comment:
you can make your code more readable just a suggestion  :smiley: 
```js 
  const userInfo = {
    username: 'fadi1', email: 'fadi1@gmail.com', password: 'asd', nativeLangId: 1, learningLangId: 1,
  };
```
and then 
```suggestion
    .then(() => addUser(userInfo))
```